redis/commands/del.cc

```diff
@@ -44,9 +45,16 @@ shared_ptr<abstract_command> del::prepare(service::storage_proxy& proxy, request
 
 future<redis_message> del::execute(service::storage_proxy& proxy, redis::redis_options& options, service_permit permit)
 {
-    return redis::delete_object(proxy, options, std::move(_key), permit).then([] {
-       return redis_message::one();
+    // FIXME: del command doesn't support to delete multiple keys once
```

Comment:
Amos, can you please separate unrelated patches to separate pull requests or mailing-list patches?
In this series, some of your patches I can commit in an instant and some I have more problems with, and I'm not sure if I should break up your pull request by committing some of the patches (technically I can, but I'm not sure I should).

Reply:
OK, I will split it to multiple PRs.